tasks/watch-css.js

```diff
@@ -14,7 +14,7 @@ governing permissions and limitations under the License.
 
 import chokidar from 'chokidar';
 import debounce from 'debounce';
-import { processCSS } from './css-tools.js';
+import { processCSS } from './build-css.js';
```

Comment:
The css-tools file only had 1 task in it so I pulled that into build-css to simplify things a bit.